packages/framework/src/servers/next.ts

```diff
@@ -5,6 +5,20 @@ import { NovuRequestHandler, type ServeHandlerOptions } from '../handler';
 import { type Either, type SupportedFrameworkName } from '../types';
 import { getResponse } from '../utils';
 
+/*
+ * Re-export all top level exports from the main package.
+ * This results in better DX reduces the chances of the dual package hazard for ESM + CJS packages.
+ *
+ * Example:
+ *
+ * import { serve, Client, type Workflow } from '@novu/framework/express';
+ *
+ * instead of
+ *
+ * import { serve } from '@novu/framework/express';
+ * import { Client, type Workflow } from '@novu/framework';
```

Comment:
```suggestion
 * import { serve, Client, type Workflow } from '@novu/framework/next';
 *
 * instead of
 *
 * import { serve } from '@novu/framework/next';
 * import { Client, type Workflow } from '@novu/framework';
```